### ISSUE TITLE
[A11yAudit] Structure Data Tables correctly - (Tables API Work)

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1334]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-7 Table
Incorrectly structured data tables

ISSUE DESCRIPTION
Some assistive technologies provide keystrokes that allow a user to interact easily with data tables. For example, screen readers allow users to navigate to each data table, and then navigate vertically and horizontally through the table providing contextual headers as the user moves through. A user can also at any time investigate the current cell where they hear the column heading, row heading, column number, row number, and value of the cell. This information is only available when the data table is marked up correctly.

Table (two headers) and 96 - Table (row header) there are "Row Heading" elements that are missing the scope="row" attribute, which would correctly associate it with the related <td> elements.

Table (span header) there are sub-header `<th>` elements present. The addition of headers attributes associated with id's are needed to ensure that all of the information within the `<table>` is properly associated with their respective headers and sub-headers.

RECOMMENDATIONS
Ensure all tabular relationships are marked up correctly. If a cell represents a row header, ensure that it has an appropriate scope="row" attribute.

Table (span header) in order to ensure the relationship between the headers and sub-headers, we recommend the use of headers associated with id's.

Note that the sub-header `<th>` elements have both headers and id attributes as they are both associated with the higher level `<th>` elements and their respective <td> elements. The `<td>` elements then point to either one or two levels of `<th>` element within their headers attributes.

**AC**

**Comment by ssabo1** [comment]
This will go with Tables API work.

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
